src/util/dsp/LogSlider.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace otto::util::dsp {
+  class LogSlider
```

Comment:
Usually i just use `struct` for everything, since it has better defaults. Then you wouldnt need that `public:` just below either. Thats just my personal style, not required